lib/restate-constructs/single-node-restate-deployment.ts

```diff
@@ -356,44 +356,22 @@ export class SingleNodeRestateDeployment extends Construct implements IRestateEn
         `roles = [`,
         `    "worker",`,
         `    "admin",`,
-        `    "metadata-store",`,
+        `    "metadata-server",`,
+        `    "log-server",`,
         `]`,
```

Comment:
Can we not set the roles explicitly? Otherwise it's gonna be problematic once we enable the http-ingress role by default because then this deployment won't run the ingress once we release 1.4.0.

Reply:
That's a reasonable thing to do! I'll update for the release.